### DISCUSSION TITLE
SVBRDF using neural networks

### DISCUSSION THREAD

**Discussion by saeedhd96** [mitsuba-renderer/mitsuba3#718] [Show and tell]
Hi,

For a project, I needed SVBRDFs whose parameters are represented using neural networks (such as albedo, roughness etc.). In the following notebook, I explained how this works:

https://github.com/inverse-neural-radiosity/inverse-neural-radiosity/blob/master/sample_scripts/tutorials/svbrdf.ipynb

Hope it is helpful for others.

**Comment by Mephisto405** [comment]
Congrats for your paper btw!

**Comment by njroussel** [comment]
Very nice!
